jupyter-book/appendix/glossary.md

```diff
@@ -46,22 +46,27 @@ is thus the black dotted line. This decision rule is used to
 the line: a [sample](#sample-instance-observation) lying on the left of the
 line will be predicted as a blue sample while a sample lying on the right of
 the line will be predicted as an orange sample. Here, we have a linear
-[classifier](#classifier) because the decision rule is defined as a line (in
+[classification model](#classification-model) because the decision rule is defined as a line (in
 higher dimensions this would be an hyperplane). However, the shape of the
 decision rule will depend on the [model](#model) used.
 
-### classifier
+### classification model
 
 A model used for [classification](#classification). These models handle
 [targets](#target-label-annotation) that contains discrete values such as
 `0`/`1` or `cat`/`dog`. For example in scikit-learn `LogisticRegression` or
-`HistGradientBoostingClassifier` are classifier
+`HistGradientBoostingClassifier` are classification model
 [classes](#target-label-annotation).
```

Comment:
Hmmm actually this label is not correct here, can you fix it in a separate PR?

classes is meant here as an OOP (Object Oriented Programming) way not a classes=labels way which is what the link goes to. The fix is just to remove the link.

Reply:
I think if you accept this suggestion it should do what I want.